causing/examples/models.py

```diff
@@ -228,21 +220,21 @@ def education():
     # note that in Sympy some operators are special, e.g. Max() instead of max()
     from sympy import Max
 
-    def define_equations(FATHERED, MOTHERED, SIBLINGS, BRKNHOME, ABILITY, AGE):
-        eq_EDUC = (
-            13
-            + 0.1 * (FATHERED - 12)
-            + 0.1 * (MOTHERED - 12)
-            - 0.1 * SIBLINGS
-            - 0.5 * BRKNHOME
-        )
-        eq_POTEXPER = Max(AGE - EDUC - 5, 0)
-        eq_WAGE = 7 + 1 * (EDUC - 12) + 0.5 * POTEXPER + 1 * ABILITY
-
-        return eq_EDUC, eq_POTEXPER, eq_WAGE
+    equations = (
+        # EDUC
+        13
+        + 0.1 * (FATHERED - 12)
+        + 0.1 * (MOTHERED - 12)
+        - 0.1 * SIBLINGS
+        - 0.5 * BRKNHOME,
+        # POTEXPER
+        Max(AGE - EDUC - 5, 0),
+        # WAGE
+        7 + 1 * (EDUC - 12) + 0.5 * POTEXPER + 1 * ABILITY,
```

Comment:
Great simplification. But how do we ensure that the equation belongs to the right yvar? The modeler needs to know the order of the yvars. But I guess that's OK.

Reply:
The same as before, we return it in the order of yvars. The old `eq_*` names were just variable names and didn't influence the behavior in any way.

Reply:
We could use a dictionary of yvar -> eq as well. I just went for the smallest change.

Reply:
It is fine as it is!